classes/Tag.php

```diff
@@ -138,6 +138,12 @@ public static function addTags($id_lang, $id_product, $tag_list, $separator = ',
     public static function updateTagCount()
     {
         if (!Module::getBatchMode()) {
+        	
+            if (!$context) {
```

Comment:
No need to check existence of $context since it's in the local scope.
It should never exist.